rancher/dashboard
# Removed required from ApiGroup feild

### Summary
Fixes #7084

Removed mandatory Asterix * sign on API Groups

### To test 
Try to create a role through Rancher >> Cluster Explorer >> RBAC >> Roles >> Create
The `API Groups` field should not have a mandatory Asterix * sign
